less/navigation.less

```diff
@@ -1 +1,37 @@
-// Navigation Styles here
\ No newline at end of file
+ img{
+        border-top: 2px dashed grey;
+        margin-bottom: 20px;
+    }
+    .navbar{
+        display: flex;
+        justify-content: space-between;
+        margin-bottom: 10px;
+        @media(max-width: 500px){
+            display: flex;
+            flex-direction: column;
+            h1{
+                text-align: center;
+                margin-bottom: 10px;
+            }
+
+            .anchor{
+                display: flex;
+                flex-direction: row;
+                justify-content: space-evenly;
+            }
+
+        }
+
+
+        a{
+            margin-right: 20px;
```

Comment:
Leaving this margin in for mobile view messes with the `justify-content` value you've specified above!